### ISSUE TITLE
Add ability to read the docker volumes from the docker sock

### ISSUE THREAD

**Issue by Akruidenberg** [Fekide/volumerize#18]
When searching for a backup tool made for docker. I found another interesting tool:

https://github.com/camptocamp/bivac

This tool reads the docker volumes directly from the docker sock. Every interval it scans for new volumes and then it can make a backup. Is a lot easier to add volumes to my backup, than adding volumes with mount points.

Is is possible to add this function to Volumerize?

**Comment by sargreal** [comment]
Yes you are right that this would be a really cool feature. However this one will probably require quite a lot of work. Additionally it will most likely not work with all backup solutions out of the box. 

I would probably like to postpone this until I do a major refactor of this repo into python. Then it will probably be much easier to implement this

**Comment by Akruidenberg** [comment]
Any progress?
